upgrade/1.4.1/README.md

```diff
@@ -0,0 +1,156 @@
+
+1. Set `CSM_RELEASE_VERSION` to the CSM release version.
+
+   ```bash
+   export CSM_RELEASE_VERSION="$(${CSM_DISTDIR}/lib/version.sh --version)"
+   echo "${CSM_RELEASE_VERSION}"
+   ```
+
+1. Download and install/upgrade the **latest** documentation on `ncn-m001`.
+
+   See [Check for Latest Documentation](../../update_product_stream/index.md#check-for-latest-documentation).
+
+## Setup Nexus
+
+Run `lib/setup-nexus.sh` to configure Nexus and upload new CSM RPM
+repositories, container images, and Helm charts:
+
+```bash
+cd "$CSM_DISTDIR"
+./lib/setup-nexus.sh
+```
+
+On success, `setup-nexus.sh` will output `OK` on `stderr` and exit with status
+code `0`. For example:
+
+```console
+./lib/setup-nexus.sh
+
+[... output omitted ...]
+
++ Nexus setup complete
+setup-nexus.sh: OK
+echo $?
+0
+```
+
+In the event of an error, consult [Troubleshoot Nexus](../../operations/package_repository_management/Troubleshoot_Nexus.md)
+to resolve potential problems and then try running `setup-nexus.sh` again. Note that subsequent runs of `setup-nexus.sh` may
+report `FAIL` when uploading duplicate assets. This is okay as long as `setup-nexus.sh` outputs `setup-nexus.sh: OK` and exits
+with status code `0`.
+
+## Upgrade services
+
+Run `upgrade.sh` to deploy upgraded CSM applications and services:
+
+```bash
+cd "$CSM_DISTDIR"
+./upgrade.sh
+```
+
+## Upload NCN images
+
+It is important to upload NCN images to IMS and to edit the `cray-product-catalog`. This is necessary when updating products
+with IUF. If this step is skipped, IUF will fail when updating or upgrading products in the future.
+
+(`ncn-m001#`) Execute script to upload CSM NCN images and update the `cray-product-catalog`.
+
+```bash
+/usr/share/doc/csm/upgrade/scripts/upgrade/upload-ncn-images.sh
+```
+
+## Update test suite packages
+
+Update the `csm-testing` and `goss-servers` RPMs on the NCNs.
+
+```bash
+pdsh -b -w $(grep -oP 'ncn-\w\d+' /etc/hosts | sort -u |  tr -t '\n' ',') 'zypper install -y csm-testing goss-servers'
+```
+
+## Verification
+
+1. Verify that the new CSM version is in the product catalog.
+
+   Verify that the new CSM version is listed in the output of the following command:
+
+   ```bash
+   kubectl get cm cray-product-catalog -n services -o jsonpath='{.data.csm}' | yq r -j - | jq -r 'to_entries[] | .key' | sort -V
+   ```
+
+   Example output that includes the new CSM version (`1.4.1`):
+
+   ```text
+   0.9.2
+   0.9.3
+   0.9.4
+   0.9.5
+   0.9.6
+   1.0.1
+   1.0.10
+   1.2.0
+   1.2.1
+   1.2.2
+   1.3.0
+   1.3.1
+   1.4.0
+   1.4.1
+   ```
+
+1. Confirm that the product catalog has an accurate timestamp for the CSM upgrade.
+
+   Confirm that the `import_date` reflects the timestamp of the upgrade.
+
+   ```bash
+   kubectl get cm cray-product-catalog -n services -o jsonpath='{.data.csm}' | yq r  - '"1.4.1".configuration.import_date'
+   ```
+
+## Complete upgrade
+
+1. Remember to exit the typescript that was started at the beginning of the upgrade.
```

Comment:
This doesn't need to be an ordered list as there is only 1 step.

Reply:
fixed